lib/opsgenie_alert.rb

```diff
@@ -0,0 +1,93 @@
+require 'httparty'
+require 'active_support/all'
+require 'active_support/time'
+
+class OpsgenieAlert
+
+  attr_reader :alert
+
+  def initialize(alert)
+    @alert = alert
+  end
+
+  def tags
+    alert["tags"]
+  end
+
+  def id
+    alert["id"]
+  end
+
+  def created_at
+    time = time_with_zone(string: alert["createdAt"])
+  end
+
+  def created_at_date
+    alert["createdAt"].to_s.split("T")[0]
+  end
+
+  def saturday?
+    created_at.saturday?
+  end
+
+  def sunday?
+    created_at.sunday?
+  end
+
+  def inhours?
+    time_range_inhours.include?(created_at.hour)
```

Comment:
This is lying. It's not in hours if it's a weekend. I'd expect this to include what you've done where you call it:
```ruby
def in_hours?
  return false if on_weekend?
  return false if on_bank_holiday?

  time_range_inhours.include?(created_at.hour)
end
```